client/keys/add.go

```diff
@@ -327,3 +336,17 @@ func printCreate(cmd *cobra.Command, info keyring.Info, showMnemonic bool, mnemo
 
 	return nil
 }
+
+func verifyMultisigTarget(kb keyring.Keyring, multisigKeys []string, newkey string) error {
+	if _, err := kb.Key(newkey); err == nil {
```

Comment:
In this case, `multisigKeys` should be key name not address. Is it right?

Reply:
yes that's right. It is derived from `flagMultisig`